deploy/scripts/src/ipv-core/test.ts

```diff
@@ -148,25 +146,99 @@ export function coreScenario1 (): void {
   sleep(Math.random() * 3)
 
   group(
-    'B01_Core_03_ClickContinueAfterLogin POST',
+    'B01_Core_03_ClickStartAfterLogin POST',
     function () {
-      const startTime = Date.now()
+      const startTime1 = Date.now()
       res = http.post(
         env.ipvCoreURL + '/ipv/page/page-ipv-identity-start',
         {
           _csrf: csrfToken
         },
         {
-          tags: { name: 'B01_Core_03_ClickContinueAfterLogin' }
+          redirects: 0,
+          tags: { name: 'B01_Core_03_ClickStartAfterLogin_01_Core' }
         }
       )
-      const endTime = Date.now()
+      const endTime1 = Date.now()
+
+      check(res, {
+        'is status 302': (r) => r.status === 302
+      })
+        ? transactionDuration.add(endTime1 - startTime1)
+        : fail('Response Validation Failed')
+
+      dcmawStubURL = res.headers.Location
+
+      const startTime2 = Date.now()
+      res = http.get(res.headers.Location,
+        {
+          tags: { name: 'B01_Core_03_ClickStartAfterLogin_02_DcmawStub' }
```

Comment:
Simplify this tag or add `// pragma: allowlist secret`

Reply:
This is done in the next commit